common/src/main/java/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationSpeechResponse.kt

```diff
@@ -0,0 +1,12 @@
+package io.homeassistant.companion.android.common.data.websocket.impl.entities
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class ConversationSpeechResponse(

```

Comment:
before merging this, we need to update the websocket response to align with the Rest response.

The Rest response has wrapped this in a dict like this https://github.com/home-assistant/frontend/blob/dev/src/data/conversation.ts#L47

Reply:
ok and that will be a change in HA core which the app will need to reflect too correct?

Reply:
yes.

Reply:
nevermind, I looked at the wrong model. this is ok.